### PR TITLE
Added default chunk size retreiver

### PR DESCRIPTION
**Pull request by KjellKod**

### CODE REVIEW

File: src/Kraken.cpp [hunk 1]
```diff
@@ -40,10 +42,17 @@ void Kraken::MaxWaitInMs(const int timeoutMs){
    mTimeoutMs = timeoutMs;
 }
 
+/// @param the new default chunk size
 void Kraken::ChangeDefaultMaxChunkSizeInBytes(const size_t bytes){
    mMaxChunkSize = bytes;
 }
 
+/// @return max chunk size
+size_t Kraken:: MaxChunkSizeInBytes() {
```
Comment (john-gress) [inline src/Kraken.cpp]: space after double colons.

File: src/Kraken.cpp [hunk 2]
```diff
@@ -118,7 +127,12 @@ Kraken::Battling Kraken::NextChunkId(){
    return Kraken::Battling::TIMEOUT;
 }
 
-/// Send data to client
+/** Send data to client
+* The actual sending of data might be send in several small chunks
```
Comment (john-gress) [inline src/Kraken.cpp]: might be *sent* ...

**Comment by john-gress** [comment]
:+1:

File: src/Kraken.cpp (hunk 1 above)
Comment (KjellKod) [inline src/Kraken.cpp]: tx. fixed this and the other comment.

**Pull request merged**